### PR TITLE
UI layouts for MVP

### LINKED ISSUE #57: Implement frontend Round pages

### LINKED ISSUE #82: Avoid displaying Player host status in the PlayerList once the game has started
Small fix that I forgot to do earlier. Once the game has started, we don't care who the host is, so why show it to the user?

### PR DESCRIPTION
**Pull request by dikunong**
Frontend layouts for SelectPunchlinePage (for The Man), EndRoundPage and EndGamePage. With these integrated into the backend, MVP is complete 😱

Note that SelectPunchlinePage currently uses a timeout hook to show the punchlines after 3 seconds. This is buggy and it reloads once, so wait 3 seconds before revealing punchlines. This will be replaced with the actual loading in of punchlines from the backend so is a non-issue for MVP.

Fully closes #57 and also incidentally closes #82 since I just did that on the way lol

### CODE REVIEW

File: frontend/src/pages/SubmitPunchlinePage/index.tsx [hunk 1]
```diff
@@ -141,7 +141,7 @@ const SubmitPunchlinePage = () => {
               handleOnClick={() => {
                 setPunchlineSubmitted(punchlineSelected);
                 setPunchlineSelected("");
-                memoryHistory.push("/postRound");
+                memoryHistory.push("/selectPunchline");
```
Comment (RawiriHohepa) [inline frontend/src/pages/SubmitPunchlinePage/index.tsx:144]: Update the path in GameRouter to reflect the name change here

**Review by RawiriHohepa — CHANGES_REQUESTED** [review]
Looks really nice and looks good running on my machine! Just a couple small changes here and there then happy to approve!

File: frontend/src/GameRouter.tsx [hunk 2]
```diff
@@ -113,15 +116,19 @@ const GameRouter = () => {
...
-          <BasicPage gameCode={gameCode} path="postRound" />
+          <EndRoundPage />
         </Route>
 
         <Route path="/scoreboard">
-          <BasicPage gameCode={gameCode} path="scoreboard" />
+          <EndGamePage />
+        </Route>
+
+        <Route path="/switchboard">
```
[the remark this answers is not in the document]
Comment (dikunong) [reply frontend/src/GameRouter.tsx:130]: I left it in just in case it was needed later on

[the remark this answers is not in the document]
Comment (RawiriHohepa) [reply frontend/src/GameRouter.tsx:130]: Yep good idea we can remove it later on

File: frontend/src/pages/SubmitPunchlinePage/index.tsx (hunk 1 above)
Comment (RawiriHohepa) [reply frontend/src/pages/SubmitPunchlinePage/index.tsx:144]: Nevermind I thought you were renaming /postRound to /selectPunchline but you were just changing where it moved to hahah

**Review by RawiriHohepa — APPROVED** [review]
Sweet those fixes look good, good job! 🎉

**Comment by taitfuller** [comment]
Nice work!

**Pull request merged**